trainTestReinforcementAlgorithm.py

```diff
@@ -0,0 +1,168 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.13.0
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %%
+import time
+import gym
+import gym_snake
+from stable_baselines3 import A2C
+from stable_baselines3.common.logger import configure
+import numpy as np
+from argparse import ArgumentParser
+from datetime import datetime
+
+
+# %% [markdown]
+# # Read This First
+# This notebook is created to make it clear how we train and test the OpenAI Gym Snake. I used jupytext so that it can be run on the server as a python commandline script or as a jupyter notebook on your local machine. If you not able to run the python file in a notebook, then you probably need to [install jupytext](https://jupytext.readthedocs.io/_/downloads/en/stable/pdf/):
+# ```
+# pip install jupytext
+# # or
+# conda install jupytext -c conda-forge 
+# ```
+
+# %% [markdown]
+# ## Train
+
+# %%
+def trainRL(
+    train_timesteps= 1000,  # Number of steps to train the snake on. One step is one action for snake.
+    env_name='snake-v0',
+):
+    env = gym.make(env_name, use_pygame=False)  # We don't want to visualize the training process
+    
+    # Model is defined here. This is hard to parameterize so change this code to play with different models
+    model = A2C('MlpPolicy', env, verbose=1)    
+    
+    t0 = time.time()
+    model.learn(
+        total_timesteps=train_timesteps  # Number of actions the model should take in learning
+    )
+    t1 = time.time()
+    print("Finished training in " + str(round(t1-t0, 2)) + " seconds")
+    
+    return model
+
+
+# %% [markdown]
+# ## Test
+# Test the model to see how well it is performing. Also have the option to visualize the result
+
+# %%
+def testRL(
+    model,
+    test_timesteps=1000,  # Number of steps to test the snake on. One step is one action for snake.
+    env_name='snake-v0',
+    visualize_testing= True,  # Set to true in order to see game moves in pygame. Should be false if run on server.
+):
+    # Setup
+    env = gym.make(
+        'snake-v0',
```

Comment:
nit: we should probably use the argument `env_name` here rather than the hardcoded one.